props/jetpack_Oli_buttons.h

```diff
@@ -0,0 +1,945 @@
+// TBA JETPACK_ALLOW_CRASHING           // if "shot at" // I could create it, if there is an interest!
+*/
+/* ============= CHECK THE BELLOW LIST TO ACTUAL BUTTON(S) COMMANDS! ============= NOT YET FINAL !!! ============= *\
+1 Button:
+POWER Button:
+
+               Activate: Short-click while OFF.
+Missile Launch Sequence: Double-click while ON or OFF.
+        Jetpack Mishaps: Hold medium-duration click while ON or OFF.
+         Activate Muted: Double-click and hold while OFF.
+        Play/Stop Music: Hold and release while OFF.
+         Turn Off Blade: Hold and wait until the blade turns off while ON.
+            Next Preset: Hold and release while OFF.
+        Previous Preset: Hold and wait while OFF.
+      Color Change Mode: Triple-click while ON.
+ Enter/Exit Volume Menu: Double-click and hold while OFF.
+              Volume Up: Short-click while in Volume Menu.
+            Volume Down: Long-click while in Volume Menu.
+          Battery Level: Triple-click while OFF.
+
+=================================================================================================================
+2 Buttons:
+POWER Button:
+
+               Activate: Short-click while OFF.
+Missile Launch Sequence: Double-click while ON or OFF.
+        Jetpack Mishaps: Hold medium-duration click while ON or OFF.
+         Activate Muted: Double-click and hold while OFF.
+        Play/Stop Music: Hold and release while OFF.
+         Turn Off Blade: Hold and wait until the blade turns off while ON.
+              Volume Up: Short-click while in Volume Menu.
+        Previous Preset: Hold and wait while OFF.
+
+AUX Button:
+
+            Next Preset: Short-click while OFF.
+      Enter Volume Menu: Long-click while OFF.
+            Volume Down: Short-click while in Volume Menu.
+          Battery Level: Hold and release while OFF.
+      Color Change Mode: Short-click while ON.
+          == Additional: Hold and release while OFF.
+
+=================================================================================================================
+3 Buttons: Same as 2 buttons except for the following:
+AUX2 Button:
+
+        Previous Preset: Short-click while OFF.
+      Color Change Mode: Short-click while ON.
+          == Additional: Hold and release while OFF.
+
+\* ============= CHECK THE ABOVE LIST TO ACTUAL BUTTON(S) COMMANDS! ============= NOT YET FINAL !!! ============= */
+
+#ifndef PROPS_JETPACK_OLI_BUTTONS_H
+#define PROPS_JETPACK_OLI_BUTTONS_H
+
+#if NUM_BUTTONS < 1
+#error Your prop NEEDS 1 or more buttons to use jetpack_prop (2 is recomended)
+#endif
+
+#include "prop_base.h"
+
+#ifdef INCLUDE_SSD1306
+#ifndef EXTRA_DISPLAY_CONTROLLER_INCLUDE
+#define EXTRA_DISPLAY_CONTROLLER_INCLUDE "props/additional_display_controllers.h"
+#endif
+#endif
+
+#define PROP_TYPE JetpackOliButtons
+
+#ifndef JETPACK_IDLE_TIME
+#define JETPACK_IDLE_TIME 1000 * 60 * 1 // Jetpack max idle mode time in millisec (default 1 min) before auto-shutdown

```

Comment:
line is long, put comment before define.
Also might be worth pointing out how this is different from IDLE_OFF_TIME